### PR TITLE
BREAKING CHANGE!: sync aerial & navic highlights with treesitter & lsp highlights

### PR DESCRIPTION
**Pull request by Isrothy**
Link Aerial and Navic highlights to treesitter and lsp highlights.
image

### CODE REVIEW

**Comment by gbprod** [comment]
Hey!
Thanks for your PR!
Does it breaks compat with neovim 0.8 ? I think I should pass #26 before.

**Comment by Isrothy** [comment]
Default semantic token values have been added

File: lua/nord/treesitter.lua [hunk 1]
```diff
@@ -111,13 +111,20 @@ function treesitter.highlights()
     ["@attribute.phpdoc"] = { fg = c.frost.artic_water },
 
     -- LSP Semantic Token Groups
+    ["@lsp.type.class"] = { link = "@type" },
     ["@lsp.type.comment"] = { link = "@comment" },
     ["@lsp.type.enum"] = { link = "@type" },
+    ["@lsp.type.enumMenber"] = { link = "@constant" },
```
Comment (gbprod) [inline lua/nord/treesitter.lua:117]: Typo : `enumMember`

**Review by gbprod — CHANGES_REQUESTED** [review]

**Comment by Isrothy** [comment]
fixed

**Review by gbprod — APPROVED** [review]

**Comment by gbprod** [comment]
thanks

**Pull request merged**